automatic/paint.net/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,21 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+Package can be verified like this:
+
+1. Go to
+
+   x32: https://www.dotpdn.com/files/paint.net.4.0.18.install.zip
+
+   to download the installer.
+
+2. You can use one of the following methods to obtain the SHA256 checksum:
+   - Use powershell function 'Get-FileHash'
+   - Use Chocolatey utility 'checksum.exe'
+
+   checksum32: F254361CAB183D637D4915B6FA321254132158D02491AD5243357868C0B07785
+
+File 'license.txt' is obtained from:
+   hhttps://www.getpaint.net/license.html
```

Comment:
@majkinetor slight typo here? `hhttps` should be `https`

Reply:
Thx. fixed